source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+.. _pdps-group-replication:
+
+********************************************************************************
+High Availability Solution with Group Replication. Technical Overview
+********************************************************************************
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+=================
+
+Every architecture and deployment depend on customer requirements and application demands for high availability and the estimated level of usage.  For example, using high read or high write applications, or both with a need for 99.999% availability.
```

Comment:
Every architecture and deployment depends

Reply:
For example, using a high read application or a high write application, or both, with 99.999% availability.

Reply:
fixed